javascript/ql/src/experimental/Security/CWE-090/LdapInjectionCustomizations.qll

```diff
@@ -16,7 +16,21 @@ module LdapInjection {
   /**
    * A data flow sink for LDAP injection vulnerabilities.
    */
-  abstract class Sink extends DataFlow::Node { }
+  abstract class Sink extends DataFlow::Node {
+    DataFlow::Node getQueryCall() {
+      exists(LdapjsClientAPICall call | result = call.getCalleeNode() |
+        this =
+          call
+              .getArgument(1)
+              .getALocalSource()
+              .(DataFlow::SourceNode)
+              .getAPropertyWrite("filter")
+              .getRhs()
+        or
+        this = call.getArgument(0)
+      )
+    }
```

Comment:
```suggestion
    /** 
     * Gets the LDAP query call that the sink flows into.
     */
    abstract DataFlow::Node getQueryCall();
```

I would suggest you move the implementation of `getQueryCall` into the concrete sink classes.  
With your current method layout it is hard to extend with a new sink that has a different `getQueryCall()` behavior. 

I would also suggest you add a qldoc to the method.